docs/adrs/cfct.md

```diff
@@ -0,0 +1,15 @@
+# Customizations for AWS Control Tower Solutions Implementation
+
+## Context
+
+When extending an AWS Control Tower Landing Zone a customization solution is required in order to e.g. deploy additional infrastructure into the Landing Zones accounts.
+
+[Customizations for AWS Control Tower](https://aws.amazon.com/solutions/implementations/customizations-for-aws-control-tower/) (CFCT) is currently the official customization solution for AWS Control Tower.
+
+## Decision
+
+- CFCT is included as an optional superwerker feature to test adoption - after adoption has been proved tighter integration of CFCT's configuration into superwerker can be explored
+- Since no configuration options are enabled initially we
```

Comment:
I would vote for simply leaving the default values in place until we decide to use CFCT (I still hope we don't need it).